rfs_test/TEST_accountservice_schema.py

```diff
@@ -29,272 +32,449 @@
+            except:
+                  print("~ \'AccountsService\' not found in the payload from GET %s" % ('/redfish/v1/AccountService'))
+
+            try: 
+                authorization = 'on'
+                json_payload, headers, status = self.http_GET('/redfish/v1/Managers/MultiBladeBMC/LogServices/Log/Entries/', rq_headers, authorization)
+                log_collection = (json_payload['Members'])
+
+            except:     
+                print("'Members\' not found in the payload from GET %s" % ('/redfish/v1/Managers/MultiBladeBMC/LogServices/Log/Entries/'));
+                return assertion_status
+         
+            attempt += 1
+
+            if attempt > authFailureThreshold:
+                log.assertion_log('line', "Assertion Failed")
+                return assertion_status
+      
+        assertion_status = log.PASS
+        log.assertion_log('line', "Assertion Passed")
+        return assertion_status
+
+    except:
+        assertion_status = log.WARN
+        log.assertion_log('line', "~ \'AccountsService\' not found in the payload from GET %s" % (
+            '/redfish/v1/AccountService'))
+        return assertion_status
+
+# end Assertion_ACCO102
+
 
 ###################################################################################################
-# Name: Assertion_ACCO103(self, log) :Account Service                                            
-# Assertion text: 
-# This property shall reference the minimum password length that the implementation will allow a 
+# Name: Assertion_ACCO103(self, log) :Account Service
+# Assertion text:
+# This property shall reference the minimum password length that the implementation will allow a
 # password to be set to.
 ###################################################################################################
 
-'''
-Step 1: Try creating an account with the password length less than the property MinPasswordLength.
-Step 2: If the value at MinPasswordLength is 0, then the assertion should automatically be passed or
-else proceed to STEP 3. 
-Step 3: Check for an error message as part of the HTTP request. 
-Step 4: Fail the assertion if an account was created sucessfully. 
-'''
+
+def Assertion_ACCO103(self, log):
+
+    log.AssertionID = 'ACCO103'
+    assertion_status = log.PASS
+    log.assertion_log('BEGIN_ASSERTION', None)
+
+    relative_uris = self.relative_uris
+    authorization = 'on'
+    rq_headers = self.request_headers()
+
+    try:
+        json_payload, headers, status = self.http_GET(
+            '/redfish/v1/AccountService', rq_headers, authorization)
+
+        MinPasswordLength = json_payload['MinPasswordLength']
+
+        if MinPasswordLength != 0:
+            rq_body = {'Name': 'Test',
+                        'Password': ''}
+
+            json_payload, headers, status = self.http_POST(
+            acc_collection + '/test', rq_headers, rq_body, authorization)
     
-## end Assertion_ACCO103
+            if status == 201:
```

Comment:
Need to double check if other codes in the 200 range are coming back in the success case.